members.html

```diff
@@ -75,12 +75,23 @@ <h1 class="subtopic">The Team</h1>
 		</section>
 
 		<section id="teamPictures" class="hcenter">
+			<!--allMembers Button-->
+			<div id="allMembersButtons" class="btn-group" role="group" aria-label="...">
+				<button type="button" class="btn btn-default btn-lg dark-button active-batch small-mobile" onclick="clickAllPresentMembers()">Present Members</button>
+				<button type="button" class="btn btn-default btn-lg dark-button large-mobile" onclick="clickAllAlumni()">Alumni</button>
+			</div><br>
+			<!--present members batchButton-->	
 			<div id="batchButtons" class="btn-group" role="group" aria-label="...">
-				<button type="button" class="btn btn-default btn-lg dark-button active-batch small-mobile" onclick="clickAll()">All</button>
-				<button type="button" class="btn btn-default btn-lg dark-button large-mobile" onclick="clickFourthYears()">Batch of 2018</button>
-				<button type="button" class="btn btn-default btn-lg dark-button large-mobile" onclick="clickThirdYears()">Batch of 2019</button>
-				<button type="button" class="btn btn-default btn-lg dark-button large-mobile" onclick="clickSecondYears()">Batch of 2020</button>
+				<button type="button" class="btn btn-default btn-lg dark-button active-batch small-mobile" onclick="clickAllPresentMembers()">All</button>
+				<button type="button" class="btn btn-default btn-lg dark-button large-mobile" onclick="clickFourthYears()">Batch of 2019</button>
+				<button type="button" class="btn btn-default btn-lg dark-button large-mobile" onclick="clickThirdYears()">Batch of 2020</button>
+				<button type="button" class="btn btn-default btn-lg dark-button large-mobile" onclick="clickSecondYears()">Batch of 2021</button>
 			</div>
+			<!--Alumni batchButton-->
+			<div id="alumniButtons" class="btn-group" role="group" aria-label="...">
+				<button type="button" class="btn btn-default btn-lg dark-button active-batch small-mobile" onclick="clickAllAlumni()">All</button>
+				<button type="button" class="btn btn-default btn-lg dark-button large-mobile" onclick="clickAllAlumni()">Alumni of 2018</button>
```

Comment:
Create a new function ```click2018Alumni()``` and set the onClick to point to that instead. This will make it easier to expand the site to include more alumni batches in the future.

Reply:
done